src/NServiceBus.Core/Transports/TransactionModeSettingsExtensions.cs

```diff
@@ -15,20 +14,13 @@ public static class TransactionModeSettingsExtensions
         public static TransportTransactionMode GetRequiredTransactionModeForReceives(this ReadOnlySettings settings)
```

Comment:
I assume this is still here because the `Receiving` property on `FeatureConfigurationContext` is internal and not public? Is there a reason to not open up access to everyone?

Reply:
Correct, I'd say we keep internal for now until the component naming scheme has stabilized? (at least we can do this in a separate PR)